students/uss_tomasz/lesson_05_lists/5_3/snakify_lesson_7_problems.py

```diff
@@ -0,0 +1,31 @@
+# https://snakify.org/lessons/lists/problems/
+from random import randint
+from collections import OrderedDict
+
+
+def remove_duplicates(list_from_user):
+    return list(OrderedDict.fromkeys(list_from_user))
+
+
+def quantity_of_bigger_than_both_neighbours(list_from_user):
+    quantity = 0
+    for i in range(1, len(list_from_user) - 1):
+        if list_from_user[i] > list_from_user[i-1] and list_from_user[i] > list_from_user[i+1]:
+            quantity += 1
+    return quantity
+
+
+list_for_test = remove_duplicates([randint(1, 10) for i in range(10)])
```

Comment:
**Super important**
Tests must be always run with the same deterministic data set. Otherwise you'll end up trying to reproduce bugs that appear in your tests once in 20 runs or worse. This is productivity killer.

Always seed your random number generator before your you it in tests.